include/eve/module/core/regular/bit_select.hpp

```diff
@@ -42,36 +42,28 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!     template< value T, value U > U bit_select)( T m, U x, U y )
-//!      noexcept;
+//!     template < value M, value T, value U > constexpr auto bit_select((Logical<M> m, T x, U y) noexcept;
```

Comment:
why not `bit_select(logical_value auto m, value auto x, value auto y)` like everywhere else ?